src/test/java/hu/blackbelt/structured/map/proxy/entity/User.java

```diff
@@ -88,5 +78,6 @@ public interface User extends Entity {
 
     void setBirthCountry(Country country);
 
+    <T> T adaptTo(Class<T> clazz);
```

Comment:
Shouldn't the `User` also extend an additional interface?

Reply:
No. Its example how to make MapHolder method's accessible.